--- origsrc/lablgtkmathview-0.7.2/Makefile.in	2005-12-24 07:33:28.000000000 -0600
+++ src/lablgtkmathview-0.7.2/Makefile.in	2008-01-09 20:49:30.937500000 -0600
@@ -4,7 +4,7 @@
 PROPCC = @OCAML_LIB_DIR@/lablgtk2/propcc
 PREFIX = 
 OBJECTS_C = ml_gtk_mathview.o
-OBJECTS = gtkMathViewProps.cmo ogtkMathViewProps.cmo gtk_mathview.cmo gtkMathView.cmo gMathView.cmo gMathViewAux.cmo
+OBJECTS = gtk_mathview.cmo gtkMathViewProps.cmo ogtkMathViewProps.cmo gtkMathView.cmo gMathView.cmo gMathViewAux.cmo
 OBJECTS_OPT = gtkMathViewProps.cmx ogtkMathViewProps.cmx gtkMathView.cmx gtk_mathview.cmx gMathView.cmx gMathViewAux.cmx
 GENERATED_FILES = gtkMathViewProps.ml ogtkMathViewProps.ml
 NULL =
@@ -66,6 +66,7 @@
 	rm -rf $(DIST_DIR)
 
 $(GENERATED_FILES): gtkMathView.props
+	$(PROPCC) $<
 
 ml_gtk_mathview.o: ml_gtk_mathview.c
 	gcc -c -I$(INCLUDEDIR) -fPIC `ocamlfind query -i-format lablgtk2` @GTKMATHVIEW_CFLAGS@ @GDOME_CFLAGS@ `ocamlfind query -i-format gdome2` $<
@@ -91,7 +92,7 @@
 $(ARCHIVE).cmxa: $(OBJECTS_OPT)
 	$(OCAMLMKLIB) -o $(ARCHIVE) $^ $(SHARED_LIBS)
 lib$(ARCHIVE).a $(DLL): $(OBJECTS_C)
-	$(OCAMLMKLIB) -o $(ARCHIVE) $^ $(SHARED_LIBS)
+	$(OCAMLMKLIB) -o $(ARCHIVE) -ldopt -L`ocamlfind printconf destdir`/lablgtk2 -ldopt -L`ocamlfind printconf destdir`/gdome2 $^ -llablgtk2 -lmlgdome $(SHARED_LIBS)
 
 $(TESTDIR)/test: $(OBJECTS_C) $(ARCHIVE).cma $(TESTDIR)/test.ml
 	mkdir -p $(TMPPKGDIR)
--- origsrc/lablgtkmathview-0.7.2/test/Makefile.in	2005-12-24 07:33:27.000000000 -0600
+++ src/lablgtkmathview-0.7.2/test/Makefile.in	2008-01-09 20:50:59.625000000 -0600
@@ -1,4 +1,4 @@
-PACKAGE = @PACKAGE@ lablgtk2.init
+PACKAGE = @PACKAGE@ lablgtk2
 MLFLAGS =
 OCAMLC = ocamlfind ocamlc $(MLFLAGS)
 OCAMLOPT = ocamlfind ocamlopt $(MLFLAGS)
